FiligranHQ/xtm-hub
# Issue/12/new homepage

# Context 
This PR reworks the homepage and the services page into a unique one. 
Notion Link : https://www.notion.so/filigran/Ability-to-see-services-in-my-homepage-10c8fce17f2a80a5ba46faacfa821e2b 
Issue Link : https://github.com/FiligranHQ/XTMHub/issues/12 

# How to test :
Connect (any kind of role to see the modification, but clearer in ADMIN role). 
See the new homepage
Subscribe to a service > The service directly changes place. 

Test on local : 
![image](https://github.com/user-attachments/assets/67cb90ab-e823-41e0-bf0e-83d7018973fc)
